BUILD

```diff
@@ -34,3 +36,10 @@ alias(
     }),
     visibility = ["//visibility:public"],
 )
+
+stardoc(
+    name = "docs",
+    input = "scad.bzl",
+    out = "scad.md",
```

Comment:
Shouldn't this be generating README.md?, I don't see scad.md being created by this commit, so I assume this wasn't executed, is that right?

Reply:
Bazel does unobvious things when a path is both a source file and a generated file. So if the README.md is to be checked in, it can't (or shouldn't) be an output.

Also, a lot  of the context in the README is not generated by this rule and trying to make it generate that would be hard compared to merging it over via any halfway decent text editor.